### PR TITLE
test: Fix Tizen test failures

### PR DESCRIPTION
**Pull request by joeyparrish**
Fixes these failures on Tizen:

Player
    rebufferGoal
      ✗ state orchestration and buffer length [Safari 3.0 (Tizen 3.0)]
  Player Src Equals
    ✗ can control trick play rate [Safari 3.0 (Tizen 3.0)]

This was not in any releases.

Reverts "fix: reBufferingGoal is not respected (#6433)"

This reverts commit 99ed5db1db35c7f838af3faf2e0b748f95638946.

Reopens #6355

### CODE REVIEW

**Review by alekzz — APPROVED** [review]
nit; IMHO it's worth to put the info about reverting another CL in the title.

**Comment by joeyparrish** [comment]
> nit; IMHO it's worth to put the info about reverting another CL in the title.

I hear you, but the titles are used to automate the generation of change logs.  A revert in the title is hard to follow in the change log.  Since the main consequence of this was fixing test failures, I put that in the title and moved the revert details to the commit message.
